clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -3480,6 +3537,20 @@ protected String getKafkaVersionTopic() {
     return kafkaVersionTopic;
   }
 
+  public boolean isStuck() {
```

Comment:
nit: isStuck is a bit too general, here we only check memory limit exceed, maybe we extend the name to be a bit more specific?

Reply:
It seems like you are registering isStuck metric..hmm, okay maybe we leave a comment to note that for now we only track memory issue for DVC at least?

Reply:
Will do.

Reply:
I agree that we should make the name more specific. Also a question for my curiosity, I think we have host-level ingestion metric so if the ingestion is stuck, customers can use that and log message to infer the situation so a new metric is not needed. Can we do that?

Reply:
No, this memory constraint stuck needs to be tracked separately since it is indicating some serious issue in DVC.
I can potentially rename it to be `isStuckByMemoryConstraint` to make it more specific.